Makefile.am

```diff
@@ -1,4 +1,9 @@
-SUBDIRS = fpmsyncd neighsyncd fdbsyncd portsyncd mclagsyncd natsyncd orchagent swssconfig cfgmgr tests gearsyncd
+if GCOV_ENABLED
+SUBDIRS = gcovpreload fpmsyncd neighsyncd portsyncd mclagsyncd natsyncd fdbsyncd orchagent swssconfig cfgmgr tests
```

Comment:
@pettershao-ragilenetworks please add `gearsyncd` here, believe this may be the reason the PR tests are consistently failing.